src/components/TextInput.tsx

```diff
@@ -102,6 +112,9 @@ function TextInput({
 				onBlur={onBlur}
 				placeholder={placeholder}
 			/>
+			{errorMessage != '' && error === true ? (
+				<ErrorMessage>{errorMessage}</ErrorMessage>
+			) : null}
```

Comment:
Here you can actually do something a bit simpler, checkout the [conditional rendering docs](https://reactjs.org/docs/conditional-rendering.html)
```suggestion
			{error && errorMessage &&
				<ErrorMessage>{errorMessage}</ErrorMessage>
			}
```

One important detail to note is:

> Note that returning a falsy expression will still cause the element after && to be skipped but will return the falsy expression. In the example below, `<div>0</div>` will be returned by the render method.
```
render() {
  const count = 0;
  return (
    <div>
      { count && <h1>Messages: {count}</h1>}
    </div>
  );
}
```
In our case the null option is `''` which in theory should be fine, otherwise making the default undefined on line 89 instead of `''`